packages/react-native/scripts/react_native_pods_utils/script_phases.sh

```diff
@@ -110,22 +103,21 @@ moveOutputs () {
 }
 
 withCodegenDiscovery () {
-    setup_dirs
-    find_node
-    find_codegen
-    generateArtifacts
-    moveOutputs
+    describe "Generating codegen artifacts"
+    pushd "$RCT_SCRIPT_RN_DIR" >/dev/null || exit 1
+        "$NODE_BINARY" "scripts/generate-codegen-artifacts.js" --path "$RCT_SCRIPT_APP_PATH" --outputPath "$RCT_SCRIPT_OUTPUT_DIR" --targetPlatform "ios"
```

Comment:
The value of `RCT_SCRIPT_OUTPUT_DIR` is always `RCT_SCRIPT_POD_INSTALLATION_ROOT`, which is set to be 
```
pushd "$PODS_ROOT/../" > /dev/null
RCT_SCRIPT_POD_INSTALLATION_ROOT=$(pwd)
```

So it will always be `Pods/..`, which is where we want codegen to be generated.
Just adding the comment for the future.